device.js

```diff
@@ -651,6 +651,30 @@ function addIndirectCorrespondents(arrOtherCosigners, onDone){
 	}, onDone);
 }
 
+function removeCorrespondentDevice(addr, onDone){
+	db.query(
+		"DELETE FROM correspondent_devices WHERE device_address=?", 
+		[addr], 
+		function(){
+			db.query(
+				"DELETE FROM device_messages WHERE device_address=?", 
+				[addr], 
+				function(){
+					onDone();
+				}
+			);
+		}
+	);
+}
+
+function readDeviceAddressesUsedInMultisigWallets(onDone){
```

Comment:
fixed; moved to wallet.js (and renamed to readDeviceAddressesUsedInSigningPaths)